### PR TITLE
Fix warning not addressed in cef191cde31cbef1da12562cda91e629eabdde93

### PR DESCRIPTION
**Pull request by trufae**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Detailed description**

...

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

**Comment by trufae** [comment]
```c
.//shlr/ar/libr_ar.a
io_private.h:4:9: warning: type of ‘io_map_new’ does not match original declaration [-Wlto-type-mismatch]
io_map.c:82:9: note: type mismatch in parameter 7
io_map.c:82:9: note: type ‘void’ should match type ‘_Bool’
io_map.c:82:9: note: ‘io_map_new’ was previously declared here
```

File: libr/io/io_private.h [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #ifndef _IO_PRIVATE_H_
 #define _IO_PRIVATE_H_
 
-RIOMap *io_map_new(RIO *io, int fd, int flags, ut64 delta, ut64 addr, ut64 size, bool do_skyline);
+RIOMap *r_io_map_new(RIO *io, int fd, int flags, ut64 delta, ut64 addr, ut64 size);
```
Comment (XVilka) [inline libr/io/io_private.h:4]: Shouldn't this be a `io_map_new` not `r_io_map_new`?

**Comment by XVilka** [comment]
```
io.c:169:93: error: too many arguments to function call, expected 6, have 7
                io_map_new (io, desc->fd, desc->perm, UT64_MAX - at + 1, 0LL, size - (UT64_MAX - at) - 1, false);
                ~~~~~~~~~~                                                                                ^~~~~
/usr/lib/llvm-6.0/lib/clang/6.0.0/include/stdbool.h:33:15: note: expanded from macro 'false'
#define false 0
              ^
./io_private.h:4:1: note: 'io_map_new' declared here
RIOMap *io_map_new(RIO* io, int fd, int perm, ut64 delta, ut64 addr, ut64 size);
^
1 error generated.
make[3]: *** [io.o] Error 1
```

**Comment by trufae** [comment]
Should be ready to go now

**Review by XVilka — APPROVED** [review]

File: libr/io/io_private.h [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #ifndef _IO_PRIVATE_H_
 #define _IO_PRIVATE_H_
 
-RIOMap *io_map_new(RIO *io, int fd, int flags, ut64 delta, ut64 addr, ut64 size, bool do_skyline);
+RIOMap *io_map_new(RIO* io, int fd, int perm, ut64 delta, ut64 addr, ut64 size);
```
Comment (ret2libc) [inline libr/io/io_private.h:4]:
```suggestion
RIOMap *io_map_new(RIO *io, int fd, int perm, ut64 delta, ut64 addr, ut64 size);
```

**Review by ret2libc — APPROVED** [review]

**Pull request merged**